### PR TITLE
Fixing disabled on $.clone

### PR DESCRIPTION
**Pull request by clrux**
@talbs This is a quick fix for the disabled states on replaced selects.

### CODE REVIEW

File: _src/pattern-library/js/select-replace.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ define([
                     }
 
                     if ($el.is(':disabled')) {
+                        console.log(replaced);
```
Comment (talbs) [inline _src/pattern-library/js/select-replace.js]: Mind removing your debug code here for production?

**Comment by talbs** [comment]
@clrux, making your commit message a bit more descriptive (like what UI element and interaction does this fix)?

**Comment by clrux** [comment]
@talbs Updated, sir!

**Comment by talbs** [comment]
:+1:

**Pull request merged**